### ISSUE TITLE
'Internationalizing Flutter apps' page issue

### ISSUE THREAD

**Issue by Pablo-Aldana** [flutter/website#5057]
Page URL: https://flutter.dev/docs/development/accessibility-and-localization/internationalization.html
Page source: https://github.com/flutter/website/tree/master/src/docs/development/accessibility-and-localization/internationalization.md

Found a typo? You can fix it yourself by going to the page source and clicking the pencil icon. Or finish creating this issue.

Description of issue:

On the section where you need to declared the localizationsDelegates the current example is listed as:

```import 'package:flutter_localizations/flutter_localizations.dart';

// ...

MaterialApp(
 localizationsDelegates: [
   // ... app-specific localization delegate[s] here
   GlobalMaterialLocalizations.delegate,
   GlobalWidgetsLocalizations.delegate,
   GlobalCupertinoLocalizations.delegate,
 ],
 supportedLocales: [
    const Locale('en', ''), // English, no country code
    const Locale('ar', ''), // Arabic, no country code
    const Locale.fromSubtags(languageCode: 'zh'), // Chinese *See Advanced Locales below*
    // ... other locales the app supports
  ],
  // ...
)
```

But it is missing the delegate for AppLocalization `  AppLocalizations.delegate,`as per the example https://github.com/flutter/website/blob/master/examples/internationalization/gen_l10n_example/lib/main.dart if this is not included it will throw a null exemption 

The final example should look like:

```
import 'package:flutter_localizations/flutter_localizations.dart';

// ...

MaterialApp(
 localizationsDelegates: [
   // ... app-specific localization delegate[s] here
   AppLocalizations.delegate,
   GlobalMaterialLocalizations.delegate,
   GlobalWidgetsLocalizations.delegate,
   GlobalCupertinoLocalizations.delegate,
 ],
 supportedLocales: [
    const Locale('en', ''), // English, no country code
    const Locale('ar', ''), // Arabic, no country code
    const Locale.fromSubtags(languageCode: 'zh'), // Chinese *See Advanced Locales below*
    // ... other locales the app supports
  ],
  // ...
)
``

**Comment by sfshaza2** [comment]
Good catch!

**Comment by sfshaza2** [comment]
Fixed by #5051
